From 33ecb7a9da19b2865922ebaade096c2a4e443e14 Mon Sep 17 00:00:00 2001
From: Mike Lundy <mike@pistoncloud.com>
Date: Tue, 8 Nov 2011 10:57:27 -0800
Subject: [PATCH] load gettext in __init__ to fix '_ is not defined'

gettext.install needs to be run on every potential entrypoint; this
resolves bug #885529 (the python standard library includes a noop
gettext library for this purpose). This is also how nova does it.

(cherry picked from commit 8f122d954bb35d18f0afb38aa2822cd1741f05f0)

Change-Id: I4ceaa4782461ebf6d27836336f288d79ba15cf2d
---
 Authors            |    1 +
 glance/__init__.py |    4 ++++
 2 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/Authors b/Authors
index 3d2ec33..7993d0b 100644
diff --git a/glance/__init__.py b/glance/__init__.py
index b606957..b6d314b 100644
--- a/glance/__init__.py
+++ b/glance/__init__.py
@@ -14,3 +14,7 @@
 #    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 #    License for the specific language governing permissions and limitations
 #    under the License.
+
+import gettext
+
+gettext.install('glance', unicode=1)
-- 
1.7.6.5

